rts/ExternalAI/Interface/AISCommands.h

```diff
@@ -258,8 +257,13 @@ enum UnitCommandOptions {
 		+ sizeof(struct SSetSizeOverlayTextureDrawerDebugCommand) \
 		+ sizeof(struct SSetLabelOverlayTextureDrawerDebugCommand) \
 		+ sizeof(struct SFeatureTraceRayCommand) \
+		+ 40 \
```

Comment:
The sum is used as a version number, so the +40 should be something larger than the size of the removed struct (to make up for the "lost" size and then some extra to bump up the version number). I went for 40 because I am reasonably certain it's larger than 4 ints regardless of architecture.

Reply:
Hmm I thought the idea was to NOT break ABI since `CMD_AISELECT` did nothing anyway.

Reply:
Makes sense, I'll change to +16. In practice I don't think any AI even looks at this version anyway.